channels.yaml

```diff
@@ -351,4 +351,9 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
-
+  - version: v1.24.13+k3s1
```

Comment:
Are we intentionally skipping `v1.24.12+k3s1` KDM?

Reply:
Yep! https://github.com/rancher/rancher/issues/40927 (I only created this stub issue in 2.7 but the same applied to 2.6)